pkg/sysregistriesv2/alias_test.go

```diff
@@ -0,0 +1,133 @@
+package sysregistriesv2
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/containers/image/v5/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAliases(t *testing.T) {
+	tmp1, err := ioutil.TempFile("", "aliases.conf")
+	assert.Nil(t, err)
+	tmp2, err := ioutil.TempFile("", "aliases.conf.lock")
+	assert.Nil(t, err)
+
+	defer os.Remove(tmp1.Name())
+	defer os.Remove(tmp2.Name())
+
+	sys := &types.SystemContext{
+		SystemRegistriesConfPath:    "testdata/aliases.conf",
+		SystemRegistriesConfDirPath: "testdata/this-does-not-exist",
+		UserAliasesConfPath:         tmp1.Name(),
+		UserAliasesConfLockPath:     tmp2.Name(),
+	}
+
+	configCache = make(map[configWrapper]*V2RegistriesConf)
+	conf, err := TryUpdatingCache(sys)
+	assert.Nil(t, err)
```

Comment:
```suggestion
	require.NoError(t)
```
an error makes `conf` `nil`, so the rest would crash without `require`. (Also elsewhere.)